projects/packages/doc-parser/composer.json

```
@@ -0,0 +1,63 @@
+{
+	"name": "automattic/jetpack-doc-parser",
+	"description": "A parser for Jetpack Monorepo to gather inline documentation.",
+	"type": "jetpack-library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"php": ">=7.0",
```

Since you're using `<rule ref="Jetpack-Compat-82" />` for phpcs, this should probably match
```suggestion
		"php": ">=8.2",
```